cypress/utils/dashboards/query_enhancement/commands.js

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+Cypress.Commands.add('waitForLoaderNewHeader', () => {
+  const opts = { log: false };
```

Comment:
I think for utilities, we need to be careful. Eventually we will move the legacy discover ftr tests back to OSD as well. So the utilities should work for both legacy and is_enhancement is true. For example if could could have both of them working, that would save us a lot of future migration efforts
```
Cypress.Commands.add('waitForLoader', (isEnhancement = false) => {
  const opts = { log: false };

  Cypress.log({
    name: 'waitForPageLoad',
    displayName: 'wait',
    message: 'page load',
  });

  cy.wait(Cypress.env('WAIT_FOR_LOADER_BUFFER_MS'));

  // Use recentItemsSectionButton for query enhancement, otherwise use homeIcon
  cy.getElementByTestId(
    isEnhancement ? 'recentItemsSectionButton' : 'homeIcon',
    opts
  );
});
```